db/migrate/20191122000001_predefined_roles.rb

```diff
@@ -20,8 +20,8 @@ def up
       )
 
       connector = Role.find_by(name: 'Connector')
-      connector.permission_grant('ticket.customer')
-
+      connector.permission_grant('ticket.agent')
+      RoleGroup.create_if_not_exists(role: connector, group: Group.find_by(name: 'Incoming'))
```

Comment:
Wouldn't it be better to search Incoming group by id in migrations in case Incoming group was created/renamed in the course of initial installation/previous update and user renamed it? I believe it should have the same id nevermind whether it was created with this name during initial installation or renamed during update.